ADAL/src/urlprotocol/ADURLProtocol.m

```diff
@@ -166,7 +165,8 @@ - (void)startLoading
         _context = context;
     }
     
-    AD_LOG_VERBOSE_F(@"-[ADURLProtocol startLoading]", context.correlationId, @"host: %@", [self.request.URL host]);
+    AD_LOG_VERBOSE(context.correlationId, nil, NO, @"%@ - host: %@", @"-[ADURLProtocol startLoading]", [self.request.URL host]);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 128 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)